eslint/eslint-typescript.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  extends: [
+    'plugin:@typescript-eslint/recommended',
+    'plugin:prettier/recommended',
+  ],
+  plugins: ['@typescript-eslint'],
+  rules: {
+    '@typescript-eslint/explicit-module-boundary-types': 'off',
+  },
+  overrides: [
+    {
+      files: ['*.js', '*.jsx'],
+      rules: {
+        '@typescript-eslint/no-var-requires': 'off',
```

Comment:
why is this rule disabled?

Reply:
Note, that it's only disabled for plain JS Files. This rule basically enforces either ESM syntax or the TS proprietary `import foo = require('foo')` for TS files targeting Node and throws if you use regular CommonJS syntax as in `const foo = require('foo')`. That's absolutely fine for TS files but not very practical if you want to write infrastructure scripts in plain JS to download translations and such.

Reply:
go for it thank you for your feedback